README.md

```diff
@@ -270,6 +272,46 @@ const allvars = env.get()
 A variable is returned by calling `env.get`. It has the exposes the following
 functions to validate and access the underlying value.
 
+#### example(string)
+Allows a developer to provide an example of a valid value for the environment
+variable. If the variable is not set (and `required()` was called), or the
+variable is set incorrectly this will be included in error output to help
+developers diagnose the error.
+
+For example:
+
+```js
+const env = require('env-var')
+
+const sampleConfig = JSON.stringify({
+  maxConnections: 10,
+  enableSsl: true
+})
+// Use POOL_SIZE if set, else use a value of 10
+const JSON_CONFIG = env.get('JSON_CONFIG')
+  .required()
+  .example(sampleConfig)
```

Comment:
This confused me a little bit when I first read it, as it looks like you’d create an object for the example when it needs to be a string. The sample config is of course stringified above, but it’s easy to miss that it’s a string being passed in.

Do you expect people to create an object and stringify it like in his example, or do you expect people to always just put string literal in the example?

Reply:
Should always be a string. You're right. Should probably make this less convoluted.

Reply:
```js
const NODE_URL = env.get('NODE_URL')
  .required()
  .example('127.0.0.1:30303')
```